<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>133</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Diameter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>160</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>NO. of parts of first circle</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>45</width>
      <height>110</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>dr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>dz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>39</y>
      <width>135</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="lineEdit1"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit2"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit3"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <widget class="QMenu" name="menuInput">
     <property name="title">
      <string>Input</string>
     </property>
     <addaction name="actionVrins"/>
     <addaction name="actionVzins"/>
     <addaction name="actionVthetains"/>
     <addaction name="actionRzmins"/>
     <addaction name="actionRmins"/>
     <addaction name="actionTheatamins"/>
    </widget>
    <widget class="QMenu" name="menuOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <addaction name="actionVrmean"/>
     <addaction name="actionVzmean"/>
     <addaction name="actionVthemean"/>
     <addaction name="actionRm"/>
     <addaction name="actionZm"/>
     <addaction name="actionThetam"/>
     <addaction name="actionNOC"/>
    </widget>
    <addaction name="menuInput"/>
    <addaction name="menuOutput"/>
    <addaction name="actionCLOSE"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionVrins"/>
   <addaction name="actionVzins"/>
   <addaction name="actionVthetains"/>
   <addaction name="actionRzmins"/>
   <addaction name="actionRmins"/>
   <addaction name="actionTheatamins"/>
   <addaction name="separator"/>
   <addaction name="actionCLOSE"/>
   <addaction name="separator"/>
   <addaction name="actionVrmean"/>
   <addaction name="actionVzmean"/>
   <addaction name="actionVthemean"/>
   <addaction name="actionRm"/>
   <addaction name="actionZm"/>
   <addaction name="actionThetam"/>
   <addaction name="actionNOC"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionVrins">
   <property name="text">
    <string>vrins</string>
   </property>
  </action>
  <action name="actionVzins">
   <property name="text">
    <string>vzins</string>
   </property>
  </action>
  <action name="actionVthetains">
   <property name="text">
    <string>vthetains</string>
   </property>
  </action>
  <action name="actionRzmins">
   <property name="text">
    <string>zmins</string>
   </property>
  </action>
  <action name="actionRmins">
   <property name="text">
    <string>rmins</string>
   </property>
  </action>
  <action name="actionTheatamins">
   <property name="text">
    <string>theatamins</string>
   </property>
  </action>
  <action name="actionCLOSE">
   <property name="text">
    <string>CLOSE</string>
   </property>
  </action>
  <action name="actionVrmean">
   <property name="text">
    <string>vrmean</string>
   </property>
  </action>
  <action name="actionVzmean">
   <property name="text">
    <string>vzmean</string>
   </property>
  </action>
  <action name="actionVthemean">
   <property name="text">
    <string>vthetamean</string>
   </property>
  </action>
  <action name="actionRm">
   <property name="text">
    <string>rm</string>
   </property>
  </action>
  <action name="actionZm">
   <property name="text">
    <string>zm</string>
   </property>
  </action>
  <action name="actionThetam">
   <property name="text">
    <string>thetam</string>
   </property>
  </action>
  <action name="actionNOC">
   <property name="text">
    <string>NOC</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
